### PR TITLE
bump version 6.6 tiup

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
bump version 6.6 tiup

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12973
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -47,7 +48,7 @@ This section introduces the preparation works needed before upgrading your TiDB
 
 ### Step 1: Review compatibility changes
 
-Review [the compatibility changes](/releases/release-6.5.0.md#compatibility-changes) and [deprecated features](/releases/release-6.5.0.md#deprecated-feature) in TiDB v6.5.0 release notes. If any changes affect your upgrade, take actions accordingly.
+Review [the compatibility changes](/releases/release-6.6.0.md#compatibility-changes) and [deprecated features](/releases/release-6.6.0.md#deprecated-feature) in TiDB v6.6.0 release notes. If any changes affect your upgrade, take actions accordingly.
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:51]: I will make the change in https://github.com/pingcap/docs/pull/12321
```suggestion
Review [the compatibility changes](/releases/release-6.5.0.md#compatibility-changes) in TiDB v6.6.0 release notes. If any changes affect your upgrade, take actions accordingly.
```

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -47,7 +48,7 @@ This section introduces the preparation works needed before upgrading your TiDB
 
 ### Step 1: Review compatibility changes
 
-Review [the compatibility changes](/releases/release-6.5.0.md#compatibility-changes) and [deprecated features](/releases/release-6.5.0.md#deprecated-feature) in TiDB v6.5.0 release notes. If any changes affect your upgrade, take actions accordingly.
+Review [the compatibility changes](/releases/release-6.5.0.md#compatibility-changes) in TiDB v6.6.0 release notes. If any changes affect your upgrade, take actions accordingly.
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:51]:
```suggestion
Review [the compatibility changes](/releases/release-6.5.0.md#compatibility-changes) and [deprecated features](/releases/release-6.5.0.md#deprecated-feature) in TiDB v6.5.0 release notes. If any changes affect your upgrade, take actions accordingly.
```

**Review by qqqdan — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**